src/main/java/aquality/selenium/configuration/driversettings/FirefoxSettings.java

```diff
@@ -33,21 +32,15 @@ public BrowserName getBrowserName() {
         return BrowserName.FIREFOX;
     }
 
-    @Override
-    protected ISettingsFile getSettingsFile() {
-        return settingsFile;
-    }
-
     private void setFirefoxPrefs(FirefoxOptions options) {
         Map<String, Object> configOptions = getBrowserOptions();
         configOptions.forEach((key, value) -> {
             if (key.equals(getDownloadDirCapabilityKey())) {
                 options.addPreference(key, getDownloadDir());
-            }else if(value instanceof Boolean){
+            } else if(value instanceof Boolean) {
                 options.addPreference(key, (Boolean) value);
-            }
-            else if (value instanceof Integer) {
-                options.addPreference(key, Integer.valueOf(value.toString()));
+            } else if (value instanceof Integer) {
+                options.addPreference(key, Integer.parseInt(value.toString()));
```

Comment:
I don't get it.
inside this foreach, value is an object.
in this section, value is instanceof Integer.
Why do we need to cast it toString() and then parse as int? Why can't we just cast it directly? Like (int) value